app/core/Engine/Engine.ts

```diff
@@ -478,6 +484,13 @@ export class Engine {
         'https://gas.api.cx.metamask.io/networks/<chain_id>/suggestedGasFees',
     });
 
+    const remoteFeatureFlagController = RemoteFeatureFlagController.init({
+      initialState,
+      controllerMessenger: this.controllerMessenger,
+      fetchFunction,
+      disabled: getBasicFunctionalityToggleState() === false
```

Comment:
Q: why testing the exact boolean false equality here?
If the value is true, it's true.
If the value is false, it's true.
If the value eis anything like undefined or null, it will be true because of the getBasicFunctionalityToggleState default.
So if we have a false here, it's necessarily a real false as a value in the state. So the following would be enough:
```suggestion
      disabled: !getBasicFunctionalityToggleState()
```
and same on line 949

Reply:
Good point Nico. Although false equality stands out to the eyes more than negation, I agree with you about simplifying this expression.

Reply:
addressed 50795f03f5cda3c5a4ad2eb42ffe680fa7c59e98